docs/usage/managed_seed.md

```diff
@@ -41,6 +41,10 @@ spec:
 
 For an example that uses non-default configuration, see [55-managed-seed-gardenlet.yaml](../../example/55-managedseed-gardenlet.yaml)
 
+### Renewing the Gardenlet Kubeconfig Secret
+
+To have the `ManagedSeed` controller renew the gardenlet's kubeconfig secret, annotate the `ManagedSeed` with `gardener.cloud/operation=renew-kubeconfig`. This will trigger a reconcile during which the kubeconfig secret is deleted and the bootstrapping is performed again.
```

Comment:
```suggestion
To have the `ManagedSeed` controller renew the gardenlet's kubeconfig secret, annotate the `ManagedSeed` with `gardener.cloud/operation=renew-kubeconfig`. This will trigger a reconciliation during which the kubeconfig secret is deleted and the bootstrapping is performed again.
```